bitcoin/CHANGELOG.md

```diff
@@ -1,3 +1,19 @@
+# 0.33.0-alpha - TODO: Set date
+
+- Enforce script size limit when hashing scripts [#2794](https://github.com/rust-bitcoin/rust-bitcoin/pull/2794)
+
+## Upgrade notes
+
+- `MAX_SCRIPT_ELEMENT_SIZE` was removed in [#2794](https://github.com/rust-bitcoin/rust-bitcoin/pull/2794)
+  you may want to use `MAX_REDEEM_SCRIPT_SIZE` or perhaps `MAX_STACK_ELEMENT_SIZE` (see `bitcoin/src/blockdata/constants`).
+- `bitcoin::ecdsa::Error` was replaced by `ecdsa::DecodeError` and `ecdsa::ParseSignatureError`
+  (returned by `ecdsa::Signature::from_slice` and `from_str` respectively).
```

Comment:
I think this should be reversed no, according with the line above.

Reply:
Huh? `from_slice` returns `DecodeError` and `from_str` returns `ParseSignatureError`.

Reply:
This is resolved by your ack, right @storopoli?

Reply:
Yes